src/db_queries.js

```diff
@@ -1,11 +1,22 @@
+const joi = require('joi');
+
 dbQueries = {};
 
+const schema = joi.object({
```

Comment:
maybe move schema inside dbQueries.postTransaction since that's the only place it's used